Resources/Prototypes/Entities/Objects/Specific/Medical/ivbag.yml

```diff
@@ -0,0 +1,195 @@
+##
+##    IV Bags & Stands
+##
+
+
+#     IV Bag
+
+- type: entity
+  parent: BaseItem
+  name: iv bag
+  id: BaseIVBag
+  # description: An injectable IV bag that slowly drips its contents.
```

Comment:
any reason why this desc is commented out?

Reply:
Currently it does not have a component able to drip its contents due to my lack of understanding of the engine.
That's the description for when they are able to do that though.